posts/how-to-integrate-aws-connect.md

```diff
@@ -0,0 +1,177 @@
+
+If you're experimenting then you'll be glad to hear that Connect is covered by the [AWS free tier](https://aws.amazon.com/connect/pricing#AWS_Free_Tier). There's a good amount of free coverage for Connect services. You shouldn't receive any charges by using a baseline setup, but as always, keep a close watch on usage so you can manage any incurred costs.
+
+## Creating and configuring the AWS Connect instance
+
+The first step is to create an AWS Connect instance. Log in to your AWS account, navigate to the AWS Connect service, and select "Create instance" from the button in the top-right corner. This will bring up a setup wizard. The default settings are good enough to get you started but adjust accordingly based on your needs. Just remember to save the administrator **username** and **password** that you configure because you will be using it later to configure your instance.
+
+![AWS Connect Instance Creation](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/05/connect-instance-creation.png)
+
+You should now have an instance created and ready with the settings that you specified through the setup process.  Once the instance has launched, you can immediately preview the Contact Control Panel ("CCP") using the following URL `https://<instance_name>.my.connect.aws/ccp-v2/`, replacing `instance_name` with your Connect instance's name.  
+
+![Contact Control Panel](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/05/connect-initial-ccp.png)
+
+The CCP (shown above) represents the managed portion of the interface provided by AWS that allows for online voice and chat. It is the cornerstone for building a streamlined communication platform.
+
+The CCP as a standalone UI has a couple drawbacks. For starters, it's another window that has to be juggled in addition to any existing applications in use by the agent. In addition, we lose real time event information that we could be leveraging to aid the agent in servicing requests.  The rest of this post focuses on how to integrate the CCP into a new or existing web application thereby gaining the full benefits of the Connect platform.
+
+## Integrating the Contact Control Panel
+
+The [Amazon Connect Streams library](https://github.com/amazon-connect/amazon-connect-streams) makes it simple to integrate your web applications with AWS Connect. It provides the functionality to directly embed the CCP to an existing (or new) web application and a robust API for hooking in to real-time events.
+
+For demonstration purposes, I created a Vue app and then fully integated the CCP to my newly created application.  The end result is a [locally-runnable demo](https://gitlab.ippon.fr/jstrickler/aws-connect-demo). The relevant pieces of the demo application are generic enough so that they can be repurposed to fit any web framework (React, Angular, etc).
```

Comment:
```suggestion
For demonstration purposes, I created a Vue.js app and then fully integrated the CCP to my newly-created application.  The end result is a [locally-runnable demo](https://gitlab.ippon.fr/jstrickler/aws-connect-demo). The relevant pieces of the demo application are generic enough so that they can be repurposed to fit any web framework (ReactJS, Angular, etc.).
```